tests/compile-fail/extern_static.stderr

```diff
@@ -0,0 +1,14 @@
+error: unsupported operation: `extern` static FOO is not supported by Miri
```

Comment:
Given such an error, how do I figure out which crate defined this static and where?
The old output handled that nicely...